index.html

```diff
@@ -28,7 +29,7 @@ <h1 id="title">Title</h1>
 <input type="checkbox" value="option3">
 <input type="checkbox" value="option4">
 <textarea > </textarea> 
-<button type="submit" id="submit">submit</button>
+<button type="submit" id="submit">Submit</button>
 <link rel="stylesheet" href="styles.css">
```

Comment:
Remove the CSS link from the form element.

Reply:
- [ ] Using the same tip as mentioned in the previous comment, it's helpful **to explain the reasoning behind your suggestion** and present it as just that - **a suggestion rather than an order**. This approach can help the student better understand **why it's beneficial to follow your advice and even learn more from it**. 😉

  _**Read more about how to write and communicate in a Code Review in this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate).**_

- [ ] **[OPTIONAL]** You could consider adding some **emojis that can add a touch of personality** and make the recommendation more engaging and visually appealing. 👌

Reply:
Great job on implementing some of the requirements suggested by the previous reviewer, However, please you didn't make your comment polite enough to the student. You can do this by adding `please` or `kindly` before making your suggestions 😄